entries/t/tolgee.io.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Tolgee": {
+    "domain": "tolgee.io",
+    "categories": [
+      "developer"
+    ],
+    "tfa": [
+      "totp"
```

Comment:
I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, please attach some screenshots of the 2FA setup process and/or login process.
> **Note** Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment, like this:
![screenshot](https://user-images.githubusercontent.com/3535780/187552643-08eea7e8-e6c6-4873-b31c-0faf4347f69f.gif)